main/lint-baseline.xml

```diff
@@ -0,0 +1,1570 @@
+            line="136"
+            column="102"/>
+        <location
+            file="res/values-uk/strings.xml"
+            line="1549"
+            column="5"/>
+    </issue>
+
+    <issue
+        id="StringFormatMatches"
+        severity="Error"
+        message="Suspicious argument type for formatting argument #2 in `number_input_title`: conversion is `s`, received `int` (argument #3 in method call) (Did you mean formatting character `d`, &apos;o&apos; or `x`?)"
+        category="Correctness:Messages"
+        priority="9"
+        summary="`String.format` string doesn&apos;t match the XML format string"
+        explanation="This lint check ensures the following:&#xA;(1) If there are multiple translations of the format string, then all translations use the same type for the same numbered arguments&#xA;(2) The usage of the format string in Java is consistent with the format string, meaning that the parameter types passed to String.format matches those in the format string."
+        errorLine1="                        .setTitle(String.format(context.getString(R.string.number_input_title), min, max))"
+        errorLine2="                                                                                                     ~~~">
+        <location
+            file="src/cgeo/geocaching/settings/ColorpickerPreference.java"
+            line="136"
+            column="102"/>
+        <location
+            file="res/values-zh-rTW/strings.xml"
+            line="1518"
+            column="5"/>
+    </issue>
+
+    <issue
+        id="StringFormatMatches"
+        severity="Error"
+        message="Suspicious argument type for formatting argument #2 in `number_input_title`: conversion is `s`, received `int` (argument #3 in method call) (Did you mean formatting character `d`, &apos;o&apos; or `x`?)"
+        category="Correctness:Messages"
+        priority="9"
+        summary="`String.format` string doesn&apos;t match the XML format string"
+        explanation="This lint check ensures the following:&#xA;(1) If there are multiple translations of the format string, then all translations use the same type for the same numbered arguments&#xA;(2) The usage of the format string in Java is consistent with the format string, meaning that the parameter types passed to String.format matches those in the format string."
+        errorLine1="                        .setTitle(String.format(context.getString(R.string.number_input_title), min, max))"
+        errorLine2="                                                                                                     ~~~">
+        <location
+            file="src/cgeo/geocaching/settings/ColorpickerPreference.java"
+            line="136"
+            column="102"/>
+        <location
+            file="res/values/strings.xml"
+            line="1972"
+            column="5"/>
+    </issue>
+
+    <issue
+        id="ImpliedQuantity"
+        severity="Error"
```

Comment:
These errors (for affected language) should be fixed implicitly after merging #10157 and incoming corresponding translations.